targets/targets.json

```diff
@@ -2373,6 +2373,7 @@
     "MBED_CONNECT_ODIN": {
         "inherits": ["MODULE_UBLOX_ODIN_W2"],
         "release_versions": ["5"],
+        "macros_add": ["MBEDTLS_CONFIG_HW_SUPPORT"],
```

Comment:
Should it not be removed, I believe?

Reply:
Yep, but if you ADD the CONFIG_HW_SUPPORT, then you are again activating it - i.e. reversing the 1st change done to the common target fíle. In my humble opinion these `macros_add` changes should not be done, we should ONLY do the common target removal.